sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```diff
@@ -119,8 +119,9 @@ studentCalculatedTotalIncome</bpmn:text>
       <bpmn:text>IN
 studentDataRelationshipStatus
 calculatedDataHasEligibleDependants
-studentDataLivingWithParents
-studentDataLivingathomeRent
+studentDataLivingAtHome
+studentDataLivingAtHomeRent
+studentDataSelfContainedSuite
```

Comment:
Minor. As the intention is to keep the text annotation up to date, the list of inputs here still doesn't seem to match the DMN inputs. 

![image](https://github.com/user-attachments/assets/ac7ff884-62a0-4944-8851-4a8f028e40d2)